lms/views/admin/application_instance.py

```diff
@@ -336,3 +362,6 @@ def _get_ai_or_404(self, consumer_key=None, id_=None) -> ApplicationInstance:
 
         except ApplicationInstanceNotFound as err:
             raise HTTPNotFound() from err
+
+    def _get_ai_link(self, ai) -> str:
+        return f"""<a href="{self.request.route_url('admin.instance.id', id_=ai.id)}">Application instance {ai.id}</a>"""
```

Comment:
A bit sad to have HTML in the backend, but without inventing something new with the flash system it seems like it's probably the best of a bad bunch.